chart/kubeapps/templates/apprepository-rbac.yaml

```diff
@@ -130,9 +131,32 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: "kubeapps:controller:apprepository-reader-{{ .Release.Namespace }}"
+  name: "kubeapps:apprepositories-read"
 subjects:
   - kind: ServiceAccount
     name: {{ template "kubeapps.apprepository.fullname" . }}
     namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: "kubeapps:apprepositories-write"
```

Comment:
I would create just one ClusterRole (this is internal, see my comment below)

Reply:
I think there's some confusion here... it's not internal. The reader cluster-role was internal, but this is renaming it so it can be both used internally for the controller, but also available to be bound in any namespace to give users the ability to read (or write - hence the second) app repos in specific namespaces. More below.